app/view/twig/editcontent/fields/_templateselect.twig

```diff
@@ -45,15 +36,26 @@
 
 {% block fieldset_controls %}
     <div class="col-sm-9">
-        <select{{ macro.attr(attributes.select) }}>
-            <option{{ macro.attr(attributes.default) }}>{{ __('(default template)') }}</option>
-            <option{{ macro.attr(attributes.divider) }}>-----------</option>
-            {% for filename, name in listtemplates(option.filter) %}
-                <option{{ macro.attr({value: filename, selected: (context.content.get(contentkey) == filename)}) }}>
-                    {{- name -}}
-                </option>
-            {% endfor %}
-        </select>
+        <div class="select-component">
+            <div>
+                <div>
+                    <select{{ macro.attr(attributes.select) }}>
+                        <option></option>
```

Comment:
Narrowed it down to this line. Tests pass without it, with it somehow the value `"on"` gets inserted.